README.md

```diff
@@ -20,10 +20,11 @@ To install the latest (stable) development version instead:
 
 ```julia
 using Pkg
+Pkg.add(PackageSpec(url="https://github.com/plotly/DashBase.jl.git"))
```

Comment:
I thought we ended up with the user not needing to explicitly add `DashBase`?

Reply:
@alexcjohnson Yes, but this is instruction for dev repo, while DashBase is not registered yet.

Reply:
OK cool - as long as it disappears once everything is registered 👍